src/mmw/apps/modeling/tasks.py

```diff
@@ -127,7 +127,14 @@ def add_huc12_source(source, srat_huc12, source_areas, total_area):
             'Area': area,
         }
 
-    def format_catchment(srat_catchment):
+    def format_catchment(srat_catchment, loads_template):
+        for key, value in srat_catchment.items():
+            if '_' in key:
```

Comment:
Consider adding a comment here to explain why we're looking for `_` in the key. Examples of what this will include and exclude could be very useful for quick reference in the future.